src/components/settings/data/service.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/prefer-default-export */
+
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+import { getConfig } from '@edx/frontend-platform/config';
+
+export const getAccessLinks = (enterpriseUUID) => {
```

Comment:
I think it might be better to add this to `/src/data/services/LmsApiService.js` for two reasons:
- Consistency, all other parts of the app use it as the source of their API routes
- Single source, keeping the service in one place would make it easier to share and update the service

Reply:
Ah, yes. Great suggestion. Will update :)